lib/web/urbandictionary.py

```diff
@@ -17,45 +17,45 @@ def __init__(self, word, url, description, example, author):
         self.example = example
         self.author = author
         self._parse()
-    
+
     def _parse(self):
-        
+
         """ Strips links from the definition and gathers them in a links property.
         """
-        
+
         p1 = "\[.*?\](.*?)\[\/.*?\]"
         p2 = "\[(.*?)\]"
         self.links = []
         for p in (p1,p2):
             for link in re.findall(p, self.description):
                 self.links.append(link)
             self.description = re.sub(p, "\\1", self.description)
-            
+
         self.description = self.description.strip()
-    
+
     def __str__(self):
-        
+
         return self.description
 
 class UrbanDictionaryError(Exception):
     pass
 
 class UrbanDictionary(list):
-    
+
     def __init__(self, q, cached=True):
-        
+
         url = "http://api.urbandictionary.com/soap?wsdl"
         key = "91cf66fb7f14bbf7fb59c7cf5e22155f"
 
-        # Live connect for uncached queries 
+        # Live connect for uncached queries
         # or queries we do not have in cache.
         cache = Cache("urbandictionary", ".pickle")
         if not cached or not cache.exists(q):
             server = soap.SOAPProxy(url)
             try:
                 definitions = server.lookup(key, q)
-            except Exception, soap.faultType:
-                raise UrbanDictionaryError, "the API is no longer supported"
+            except Exception(soap.faultType):
```

Comment:
I haven't seen this before, is it meant to be `except (Exception, soap.faultType):` ?